modules/tfm/tfm/boards/common/attest_hal.c

```diff
@@ -102,20 +263,8 @@ enum tfm_plat_err_t tfm_plat_get_boot_seed(uint32_t size, uint8_t *buf)
 
 enum tfm_plat_err_t tfm_plat_get_implementation_id(uint32_t *size, uint8_t *buf)
 {
-	enum tfm_plat_err_t err;
-	size_t otp_size;
-
-	err = tfm_plat_otp_read(PLAT_OTP_ID_IMPLEMENTATION_ID, *size, buf);
-	if (err != TFM_PLAT_ERR_SUCCESS) {
-		return err;
-	}
-
-	err = tfm_plat_otp_get_size(PLAT_OTP_ID_IMPLEMENTATION_ID, &otp_size);
-	if (err != TFM_PLAT_ERR_SUCCESS) {
-		return err;
-	}
-
-	*size = otp_size;
+	*size = 64;
```

Comment:
size is 32 in the attestation token spec.

Reply:
https://www.ietf.org/archive/id/draft-tschofenig-rats-psa-token-10.html#sec-implementation-id

Reply:
Another magic constant. Could this function return the number of bytes read, similar to tfm_plat_otp_get_size.

Reply:
If we are reading the spec. correctly and it must be 32 bytes then I think it best to hardcode the number 32 so it is clear to the reader that this is a constant.

Reply:
(we can add a comment I guess to make it clear to the reader that we are not unnecessarily hardcoding something)

Reply:
Added define.